LambdaRuntimeDockerfiles/Images/net6/amd64/Dockerfile

```diff
@@ -69,4 +63,10 @@ ENV \
 COPY --from=builder-net6    /dotnet         ${DOTNET_ROOT}
 COPY --from=publish         /app/publish    ${LAMBDA_RUNTIME_DIR}
 
+# Generate runtime-release file
+RUN export BUILD_TIMESTAMP=$(printf '%x' $(date +%s)) && \
+    export LOGGING_PROTOCOL="LOGGING=amzn-stdout-tlv" && \
+    export LAMBDA_RUNTIME_NAME="LAMBDA_RUNTIME_NAME=${LAMBDA_RUNTIME_NAME}" && \
+    echo -e "NAME=dotnet\nVERSION=${ASPNET_VERSION}-${BUILD_TIMESTAMP}\n${LOGGING_PROTOCOL}\n${LAMBDA_RUNTIME_NAME}\n" > ${LAMBDA_RUNTIME_DIR}/runtime-release
```

Comment:
Should L70 use `DOTNET_VERSION` instead of `ASPNET_VERSION`?

Building this branch locally but falling back to 6.0.11, for .NET6 + amd64 I'm still ending up with
```
bash-4.2# cat /var/runtime/runtime-release
NAME=dotnet
VERSION=6.0.13-63c6ca55
LOGGING=amzn-stdout-tlv
LAMBDA_RUNTIME_NAME=dotnet6
```
Though `DOTNET_VERSION` is set correctly:
```
bash-4.2# printenv
DOTNET_RUNNING_IN_CONTAINER=true
HOSTNAME=88ce0937b2a4
DOTNET_CLI_TELEMETRY_OPTOUT=true
TERM=xterm
DOTNET_NOLOGO=true
DOTNET_VERSION=6.0.11
LAMBDA_TASK_ROOT=/var/task
LD_LIBRARY_PATH=/var/lang/lib:/lib64:/usr/lib64:/var/runtime:/var/runtime/lib:/var/task:/var/task/lib:/opt/lib
PATH=/var/lang/bin:/usr/local/bin:/usr/bin/:/bin:/opt/bin
PWD=/var/task
DOTNET_ROOT=/var/lang/bin
LAMBDA_RUNTIME_DIR=/var/runtime
LANG=en_US.UTF-8
TZ=:/etc/localtime
SHLVL=1
HOME=/root
_=/usr/bin/printenv
```

Reply:
User error - I was rebuilding the net5 image and then verifying an old image of .NET 6.